kernel/kernel-api/src/main/java/io/delta/kernel/internal/tablefeatures/TableFeatures.java

```diff
@@ -39,6 +40,9 @@
 /** Contains utility methods related to the Delta table feature support in protocol. */
 public class TableFeatures {
 
+  /** The prefix for setting an override of a feature option in the */
```

Comment:
`in the` -- missing end of sentence

Reply:
does setting this override just mark it as supported (in the protocol) or do we also then go and **eable** it via any optional table properties, too?